.github/workflows/backend-ci.yml

```
@@ -0,0 +1,108 @@
+    runs-on: ubuntu-latest
+    
+    defaults:
+      run:
+        working-directory: ./backend
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x]
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)